db/migrate/20210808180355_add_seats_to_cinema_hall.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class AddSeatsToCinemaHall < ActiveRecord::Migration[6.1]
+  
   def change
     add_column :cinema_halls, :seats, :text, array: true, default: []
```

Comment:
Why such a structure? I think it might cause you a lot of problems for example how would change or remove one seat from the cinema_hall because of renovation? 

In the schema file [Slack](https://monterail.slack.com/archives/C02A0T92BMW/p1628151618098600) you can see that seats have separate table with association to the cinema hall.

Reply:
btw. just wanted to explain - we discussed during data modelling session this problem and we decided that everyone will create code that will consider a good option and then we will discuss pros and cons of all solutions. :)